Updating readmes, package.json and adding url as input

jkarimi91 opened the pull request:
* updating `maintainers.md` to be a little more useful i.e. it gives a sense of the underlying model/design without having to sink time into reading the papers and looking at the code.
* adding input/output examples to `package.json`
* removing websites from `package.json'
* adding link to `maintainers.md`/contribution section to `readme.md` making it a bit more obvious that the apps are opensource/can be contributed to.
* adding url as input (not just base64 encoded img)

jay-karimi committed:
Removing website

jay-karimi committed:
Updating maintainers

jay-karimi committed:
Making accuracy sources clear

jay-karimi committed:
Adding institutions to maintainers

jay-karimi committed:
Accepting image url as input

jay-karimi committed:
Adding input/output examples

jay-karimi committed:
Adding link to maintainers

bvallelunga commented on app/main.py:
```diff
@@ -21,10 +23,12 @@ def __init__(self, model=None):
         if 'image' not in input:
             raise KeyError("No key named 'image' in input.")
-        try:
+        if is_base64_str(input['image']):
```
Change it to have 2 separate inputs:
- `image_url` xor `image_base64`

Have validation check to see if it passes the XOR (ie: one must exist but not both).

jay-karimi committed:
Splitting input key into two seperate keys

`ModelInterface` no longer accepts an `image` key; the input key
must either be `image_base64` or `image_url`. This allows us to
handle the two input types seperately and, as a result, in a
cleaner fashion.

jkarimi91 commented:
@bvallelunga per your comment, I updated the PR so that the input key is now 2 separate input keys i.e. `image` is now `image_base64` and `image_url`. Take a look when you have a chance and let me know what you think.
